masonry/src/render_root.rs

```diff
@@ -492,10 +509,7 @@ impl RenderRoot {
         // tree needs to be rebuilt. Usually both happen at the same time.
         // A redraw will trigger a rebuild of the accessibility tree.
         // TODO - We assume that a relayout will trigger a repaint
-        if self.root_state().needs_paint
-            || self.root_state().needs_accessibility
-            || self.root_state().needs_layout
-        {
+        if self.root_state().needs_paint || self.root_state().needs_accessibility {
```

Comment:
I guess `needs_layout` is not necessary because `needs_paint` (and `needs_accessibility`) is implicitly set in `run_layout_on` (and it's planned to detect whether a layout pass really needs a rerender)?

Reply:
`needs_layout` is not necessary anymore because it's one of the rewrite passes. So if `needs_layout` is true, the check above will have triggered and requested a new frame.